### ISSUE TITLE
building on debian testing/unstable

### ISSUE THREAD

**Issue by pniederlag** [utsushi/utsushi#17]
After my new scanner arrived today I was really upset by learning that official epson packages for debian wont't work on debian testing/unstable.

So i spent half a day fiddling with various ways to rebuild imagescan on my debian machine (git-dpm, git-buildpackage, ...).  As I have seen @paddy-hack you are as well maintainer of the debian/ series of patches? It would be awesome if you could ease the way of getting utsushi/imagescan working on debian testing/unstable.

Would you be able to add debian/ files on this repository?

**Comment by pniederlag** [comment]
the purpose and workflow of the two different repositories "utsushi" and "imagescan" is still not clear to me. What's your suggestion for a user that has troubles using the official binaries from epson, which repository has better chance to help the user to get a working utsushi/imagescan binary?

**Comment by pniederlag** [comment]
the utsushi repository seems to carry the improvements over the imagescan repository.

So basically imagescan is "upstream" and utsushi is "maintainrs versions". If you need to rebuild the packages yourself you should probably rather head out to the utsushi repository as compilation is more straight forward than on upstream.

At least that's my current state of knowledge. ;)

**Comment by paddy-hack** [comment]
The `imagescan` project really only tracks the "Vendor Upstream".  This is documented on the [project's "home page"](http://utsushi.github.io/imagescan/) and I just saw that that needs an update :wink:.  The `utsushi` project can be thought of as the "Community Code Base".

If the "official" binary packages from the [EPSON Download Center](http://download.ebz.epson.net/dsc/search/01/search/?OSC=LX) work for you, that is probably the easiest route.  If not, then you can try [contacting upstream](mailto:linux-scanner@epson.jp) or head over to the `utsushi` project here and see if there is anyone willing and able to help you out.

Now, as to getting `utsushi` to run on Debian's `testing/unstable`, that's a bit of a tough one because it is a moving target.  While I use the same on my own machines, I do most of my development efforts in [Docker](https://www.docker.com/) containers of a more `stable` nature.  However, once #15 gets taken care of, I expect no problems building on today's Debian `testing/unstable`.  But who knows what tomorrow will bring :wink:.

Finally, a `debian/` directory?  I'm not sure _I_ want to go there for Utsushi.  There are just too many distributions out there.  I had much rather that the various distributions start packaging Utsushi themselves and, if they want to, they can get a project here to publish their efforts in any way they see fit.

**Comment by pniederlag** [comment]
well, it's just an unfortunate mess for me... my hope/wish was for a dpkg -i somefreaking.deb until I learned that upstream(Epson) is building only on stable. :(

Since then I am trying to get hold of a build for debian testing/unstable. As my knowledge is limited in regards to c/c++ I am kind of lost on all these build related things.

Currently I managed to get some kind of "imagescan" binary working mostly stable on my machine, so I'll probably just stick with it for now.
